src/grammar/__snapshots__/grammar.spec.ts.snap

```diff
@@ -211,11 +211,22 @@ Line 1, col 14:
 `;
 
 exports[`grammar should fail case-22 1`] = `
-"<unknown>:1:4: Syntax error: expected not an idPart 
-Line 1, col 4:
-> 1 | function a(): Int { 
-         ^
-  2 |     return 42;
+"<unknown>:6:14: Empty parameter list should not have a dangling comma.
+Line 6, col 14:
+  5 | fun b(): Int {
+> 6 |     return a(,);
+                   ^
+  7 | }
+"
+`;
+
+exports[`grammar should fail case-23 1`] = `
+"<unknown>:7:1: Abstract function doesn't have abstract modifier
```

Comment:
Looks like a different error message than I would have expected

Reply:
Missed that, sorry! Fixed.